FirebaseAuth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
```

Comment:
Will be 6.7.0 for sure

Reply:
Discussed offline - will leave it as is until it's released.